tests/SprykerTest/Zed/Testify/_support/Helper/Business/BusinessHelper.php

```diff
@@ -50,6 +50,11 @@ class BusinessHelper extends AbstractHelper
      */
     protected const SHARED_FACTORY_CLASS_NAME_PATTERN = '\%1$s\Shared\%3$s\%3$sSharedFactory';
 
+    /**
+     * @var string
+     */
+    protected const BUSINESS_LAYER_NAME = 'Zed';
```

Comment:
Layers are used in Zed we have `Business`, `Communication`, etc. Rename to `APPLICATION_NAME`